knowledge-base/displaying-telerik-report-in-net-maui-and-maui-blazor-applications.md

```diff
@@ -0,0 +1,197 @@
+
+1. Firstly, you need to create the .NET Maui app:
+2. Then you can directly add the pure Html5 Report Viewer through a Web View. For example:
+
+	````HTML
+<WebView HeightRequest="600" WidthRequest="800">
+		<WebView.Source>
+			<HtmlWebViewSource>
+				<HtmlWebViewSource.Html>
+					<![CDATA[
+	<html xmlns="http://www.w3.org/1999/xhtml">
+		<head>
+			<title>Telerik HTML5 Report Viewer Demo</title>
+			<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+			<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+			<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+			<link href="https://kendo.cdn.telerik.com/2022.1.301/styles/kendo.common.min.css" rel="stylesheet" id="common-css" />
+			<link href="https://kendo.cdn.telerik.com/2022.1.301/styles/kendo.blueopal.min.css" rel="stylesheet" id="skin-css" />
+			<script src="https://demos.telerik.com/reporting/api/reports/resources/js/telerikReportViewer"></script>
+			<style>
+				body {
+					font-family: Verdana, Arial, sans-serif;
+					margin: 5px;
+				}
+			
+				#reportViewer1 {
+				. . .
+				}
+			</style>
+		</head>
+		<body>
+
+			<div id="reportViewer1">
+				loading...
+			</div>
+
+			<script type="text/javascript">
+				$(document).ready(function () {
+					$("#reportViewer1")
+						.telerik_ReportViewer({
+							serviceUrl: "https://demos.telerik.com/reporting/api/reports/",
+							reportSource: {
+								//parameters: {}
+								//parameters: { Year: [2001, 2003, 2004] }
+							},
+							// Report Server connection configuration
+							// If Report Server is used instead of hosting a REST Service, comment out 'serviceUrl' and 'reportSource' above
+							// uncomment 'reportServer' and 'reportSource' below
+							//reportServer: {
+							//    url: "http://report-server-host:83",
+							//    username: "admin",
+							//    password: "adminpass"
+							//},
+							//reportSource: {
+							//    // For Report Server, use "{Category}/{ReportName}"
+							//    report: "Samples/Dashboard"
+							//},
+						});
+				});
+			</script>
+
+		</body>
+	</html>
+	]]>
+				</HtmlWebViewSource.Html>
+			</HtmlWebViewSource>
+		</WebView.Source>
+	</WebView>
+````
+
+You can also put the page of the viewer into a separate html file and refer it into the web view.
+
+## Embedding Telerik Reporting into a .NET MAUI Blazor App 
+
+For this example, we will use the the Blazor Report Viewer and the approach from the How to Use Blazor Report Viewer. 
+You may also check our Native Blazor Report Viewer released in R3 2022. An example can be found in the installation
+folder of Telerik Reporting: C:\Program Files (x86)\Progress\Telerik Reporting Version\Examples\CSharp\CSharp.BlazorNativeExample.VS2022.sln.
+
+ 1. Create the Maui Blazor app:
+ 2.  Add the Telerik.ReportViewer.Blazor NuGet package
+ 3.  In the wwwroot -> index.html, add the following scripts 
```

Comment:
You may put the file names in ``, so that they appear highlighted. This may be done for the whole article.